kestrel-opentelemetry-metrics/src/main/kotlin/com/dreweaster/ddd/kestrel/application/reporting/opentelemetry/traces/OpenTelemetryTraceDomainModelReporter.kt

```diff
@@ -0,0 +1,131 @@
+package com.dreweaster.ddd.kestrel.application.reporting.opentelemetry.traces
```

Comment:
I've split out tracing as it's a separate concert. It's also not currently used, or alerted on.